### DISCUSSION TITLE
Your account is currently locked from purchases. Please update your payment information.

### DISCUSSION THREAD

**Discussion by lnh2023** [community/community#129477] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi,

Please can you help me to fix the issue:
When I click button: **Get access to Github Copilot** and than click button **Submit** to **Confirm your payment details** is appearing the message **Your account is currently locked from purchases. Please update your payment information.**

I'm from **Kosovo**, and our country is not in list of countries, can you fix please?

With respect,
Lindita Nebiu Hyseni

![image](https://github.com/community/community/assets/121866495/ae4da7b2-f399-4ee0-801f-1752fa9500c2)

**Comment by Akash1134** [comment]
Hi USERNAME! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by mamlzy** [reply]
no answer from github support, tho. so how do i fix this? its been 3 days

**Reply by wangchuan2008888** [reply]
same to me, and its been 7 days.
